django/Dockerfile

```diff
@@ -1,21 +1,9 @@
-FROM python:3-buster as build
-
-RUN mkdir -p /install
-COPY requirements.txt /
-RUN pip install --no-warn-script-location --prefix=/install -r /requirements.txt
-
-
-
-FROM python:3-slim-buster
-
-ENV PYTHONDONTWRITEBYTECODE 1
```

Comment:
You should leave this `ENV` line in.

Reply:
I added it back in. Also learned something :)

Reply:
It makes the image smaller. Now, using gunicorn & uvicorn, there will be multiple worker processes, and each will compile its own bytecode, which will be a slight performance hit on initial load, but I think it's fine? Worth testing image size vs. performance later.